packages/sdks/package.json

```
@@ -41,5 +40,9 @@
     "commitizen": {
       "path": "git-cz"
     }
+  },
+  "resolutions": {
+    "@X/X: mitosis line below is removed in CI to allow for proper usage of Mitosis.": "0.0.1",
```

Does it have to have that syntax? How about something closer to actual comments?
```suggestion
    "// NOTE: mitosis line below is removed in CI to allow for proper usage of Mitosis.": "0.0.1",
```

---

Yeah, it needs to confuse the key with an actual module name. I had your suggestion first and running `yarn` ends up deleting the line 😄.